src/execution_plan/ops/op_update.c

```
@@ -49,14 +52,18 @@ static void freeCallback
 static dictType _dt = { _id_hash, NULL, NULL, NULL, NULL, freeCallback, NULL,
 	NULL, NULL, NULL};
 
-OpBase *NewUpdateOp(const ExecutionPlan *plan, rax *update_exps) {
+OpBase *NewUpdateOp
+(
+	const ExecutionPlan *plan,
```

cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)